tests/hazmat/primitives/test_scrypt.py

```diff
@@ -77,6 +97,7 @@ def test_password_not_bytes(self, backend):
 
     @pytest.mark.parametrize("params", vectors)
     def test_verify(self, backend, params):
+        _skip_if_memory_limited(backend._scrypt_mem_limit, params)
```

Comment:
I'm not wild about the tests consuming a non-public API that's not in the interface. Is there a reason not to just expose a `_MAX_MEMORY` constant in the `scrypt` module and have both use it?